ckanext/qdes_schema/templates/package/metadata.html

```diff
@@ -28,7 +28,7 @@ <h3 class="pull-left">{{ _('Metadata') }}</h3>
         {% endif %}
         <li>
           <a href="{{ h.url_for('dataset.read', id=pkg_dict['name']) }}.ttl"
-             target="_blank" download><span>RDF / TTL</span></a>
+             target="_blank"><span>RDF / TTL</span></a>
```

Comment:
Why is this being removed? I thought we wanted to force the browser to download the file?

Reply:
just for testing